docs/en/vrf.md

```diff
@@ -46,41 +53,41 @@ For convenience, Nile TestNet has deployed `WinkMid` contract and encapsulated t
   :::
 
 ## Node Deployment
-For node deployment, please refer to [WINkLink Node Deploy Doc](https://docs.winklink.org/v1/doc/en/deploy.html) .  This section only lists the differences of VRF node deployment.
+For node deployment, please refer to [WINkLink Node Deploy Doc](https://doc.winklink.org/v1/doc/en/deploy.html) .  This section only lists the differences of VRF node deployment.
 
-WINkLink node can be deployed after the VRFCoordinator contract is deployed.
+WINkLink node should be deployed after the VRFCoordinator contract is deployed.
 
-WINkLink node (project directory `node`) code is available at: <https://github.com/wink-link/winklink/tree/feature/vrf/node>.
+WINkLink node (project directory `node`) code is available at: <https://github.com/wink-link/winklink/tree/feature/rename2wink/node>.
 
-After compilation, `node-v1.0.jar` will be stored in `node/build/libs/` under the project source code directory.
+After compilation, `node-v1.0.jar` will be stored in `node/build/libs/` under the project root directory.
 
 ### Node Configuration
 
-After the node configuration file is confirmed, it is required to create a `vrfKeyStore.yml` file and write the private key for VRF (support multiple VRF private keys in one node):
+After the node configuration file is confirmed, it is required to create a `vrfKeyStore.yml` file and set the private key for VRF (support multiple VRF private keys in one node):
 
 ```text
 privateKeys:
-  - *****(32字节 hex 编码私钥)
+  - *****(private key in hexadecimal format)
 ```
 
-Support to dynamically update vrfkeystore without restarting the node server. The steps are:
+Support for dynamically updating vrfkeystore without restarting the node server. The steps are:
 
 First, add a new VRF private key to the `vrfKeyStore.yml` 
 
 Second, execute the following command:
 
 ```sh
-curl --location --request GET 'http://localhost:8080/vrf/updateVRFKey/vrfKeyStore.yml'
+curl --location --request GET 'http://localhost:8081/vrf/updateVRFKey/vrfKeyStore.yml'
 ```
 
 ::: tip
-It is for important safety concern that you use files to provide private information instead of command line. In the production environment, set the permission of the private file `vrfKeyStore` as 600, meaning that only the owner can read or write the data.
+It is for important safety concern that you use file to provide private information instead of command line. In the production environment, set the permission of the private file `vrfKeyStore` as 600, meaning that only the owner can read or write the data.
```

Comment:
It is for important safety concerns that you use files to provide private information instead of the command line. In the production environment, set the permission of the private file `vrfKeyStore` as 600, meaning that only the owner can read or write the data.